dttools/src/libforce_halt_enospc.c

```diff
@@ -55,6 +48,21 @@ int open(const char *path, int flags, ...)
 	fd = original_open(path, flags, mode);
 
 	if(fd == -1 && errno == ENOSPC) {
+		int err_fd;
+		char *filename;
+		char *parent = "../";
```

Comment:
I would recommend to have the default name as:

cctools_disk_allocation_exhausted.PID (where PID comes from getpid()).

If CCTOOLS_DISK_ALLOC is defined, then that name should not be modified.